guide/onboarding/creating-a-new-wallet.md

```diff
@@ -0,0 +1,84 @@
+
+{% include /tip/open.html label="Tip: Break onboarding up" icon="info" color="blue" %}
+
+The wallet creation process is an excellent place to break things up into different activities, events, and tasks. For example, you might want to dedicate multiple screens to generate a recovery phrase and explain why it is essential.
+
+{% include /tip/close.html %}
+
+# Creating a new wallet
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/onboarding/intro-screen.jpg"
+   retina = "/assets/images/guide/onboarding/intro-screen@2x.jpg"
+   alt-text = "Onboarding intro screen"
+   layout = "float-left-desktop -background -shadow"
+   caption = "From the [Onboarding Prototype](https://www.figma.com/proto/oLJlzjXqgoU7efgXJM8FlK/Onboarding-Prototypes?node-id=1%3A4&viewport=332%2C345%2C0.5653080344200134&scaling=min-zoom&page-id=0%3A1)"
+%}
+
+After downloading a wallet application, users are typically presented with the option of either creating a new wallet or restoring an existing one. Obviously, first-time users will be going through the flow of creating a new wallet.
+
+Once an individual selects “create a new wallet”, it’s common for beginner friendly applications to first present informational carousels that provide a high level overview on the implications, benefits, and responsibility that comes along with having a non-custodial wallet. This moment is a great place to explain that, unlike traditional centralized financial products, a user’s “account” information cannot be recovered by the respective company. Rather, it is up to the user to safely protect their wallet information.
+
+</div>
+
+{% include /tip/open.html label="Don’t: Hide the implications of a non-custodial wallet" icon="forbid" color="red" %}
+
+Within your onboarding process, be clear about what it means to have a non-custodial wallet. (and the responsibility that comes along with it). Within your onboarding process, be clear that “account” information cannot be recovered by your respective team.
+
+{% include /tip/close.html %}
+
+While this may sound daunting to first-time users, it’s crucial that these caveats are not hidden. Doing so may severely compromise the safety of their funds, as their default assumptions and behaviors could potentially mimic how they typically use centralized applications (for example, feeling like it is OK if they forget their account information because they can contact customer support to help them recover it). According to a study conducted by Ekandari et al about the usability of cryptocurrency wallets, it became clear that users struggled greatly with technical terminology and feeling as though they had a lack of guidance during wallet setup. By walking users through a short set of carousels that clearly frame these Bitcoin-specific features, users will feel well-equipped to confidently navigate and use your wallet.
```

Comment:
```suggestion
While this may sound daunting to first-time users, these caveats mustn't be hidden. Doing so may severely compromise their funds' safety, as their default assumptions and behaviors could potentially mimic how they typically use centralized applications (for example, feeling like it is OK if they forget their account information because they can contact customer support to help them recover it). According to a study conducted by Ekandari et al about the usability of cryptocurrency wallets, it became clear that users struggled greatly with technical terminology and feeling as though they had a lack of guidance during wallet setup. By walking users through a short set of carousels that clearly frame these Bitcoin-specific features, users will feel well-equipped to confidently navigate and use your wallet.
```